src/main/resources/org/jenkinsci/plugins/codescene/CodeSceneBuildAction/summary.jelly

```diff
@@ -158,6 +158,8 @@
                             </ul>
                         </j:if>
 
+                        <p>${entry.description}</p>
```

Comment:
Perhaps this should be at the very top of the summary page? 
This is how it looks like now: 
![image](https://user-images.githubusercontent.com/1083629/47722452-bd877400-dc52-11e8-95a6-308e7d0754ed.png)